page/plugins/advanced-plugin-concepts.md

```diff
@@ -240,10 +240,14 @@ A bad implementation:
 
 ```
 // Plugin code
-$( "<div id=\"the-gallery-wrapper\" />").appendTo( "body" );
+$( "<div class=\"gallery-wrapper\" />").appendTo( "body" );
```

Comment:
This example doesn't really use the [code style guide](http://contribute.jquery.org/style-guide/js/). Since you're already modifying this line, could you handle it on the while? Two things:

* Use single quotes inside of the doubles, instead of escaping.
* There must be a space right before the closing param.

Thanks!